tests_peerscout_api/unit_tests/main_test.py

```diff
@@ -200,7 +228,8 @@ def test_should_have_recomendation_heading_when_the_recomendation_avaliable(
             assert heading in get_recommendation_html(
```

Comment:
I think I see now. Here it might be preferable if we resolve the placeholder and check that it has been resolved by `get_recommendation_html`, providing it with an example "editor type".

Reply:
sure I will try it now